docker-entry

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+sleep 10
+
+echo "Starting DB maintenance"
+
+for f in /var/opt/psql/scripts.d/*.sql; do
+  echo "Running $f"
+  psql -f "$f"
```

Comment:
Please `-a` to the args so it will print out the queries and results so they show up in logs.